### PR TITLE
Add integration test for table_aws_config_conformance_pack closes #241

### LINKED ISSUE #241: Fix column names and integration test for table_aws_config_conformance_pack
**Describe the bug**
Column names were not as per the standard column names and integration test terraform plan was wrong

**Steampipe version (`steampipe -v`)**
Example: v0.3.0

**Plugin version (`steampipe plugin list`)**
Example: v0.5.0

**To reproduce**
Steps to reproduce the behavior (please include relevant code and/or commands).

**Expected behavior**

**Additional context**

### PR DESCRIPTION
**Pull request by Paulami30**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
No env file present for the current environment:  staging 
 Falling back to .env config
No env file present for the current environment:  staging
customEnv TURBOT_TEST_EXPECTED_TIMEOUT undefined

SETUP: tests/aws_config_conformance_pack []

PRETEST: tests/aws_config_conformance_pack

TEST: tests/aws_config_conformance_pack
Running terraform
data.aws_region.alternate: Refreshing state...
data.aws_region.primary: Refreshing state...
data.aws_caller_identity.current: Refreshing state...
data.aws_partition.current: Refreshing state...
data.null_data_source.resource: Refreshing state...
aws_iam_role.r: Creating...
aws_iam_role.r: Still creating... [10s elapsed]
aws_iam_role.r: Creation complete after 18s [id=awsconfig-example]
aws_config_configuration_recorder.configuration_recorder: Creating...
aws_config_configuration_recorder.configuration_recorder: Creation complete after 8s [id=turbottest37108]
aws_config_conformance_pack.named_test_resource: Creating...
aws_config_conformance_pack.named_test_resource: Still creating... [10s elapsed]
aws_config_conformance_pack.named_test_resource: Still creating... [20s elapsed]
aws_config_conformance_pack.named_test_resource: Still creating... [30s elapsed]
aws_config_conformance_pack.named_test_resource: Creation complete after 39s [id=turbottest37108]

Warning: Deprecated Resource

The null_data_source was historically used to construct intermediate values to
re-use elsewhere in configuration, the same can now be achieved using locals


Apply complete! Resources: 3 added, 0 changed, 0 destroyed.

Outputs:

account_id = 352685002396
aws_partition = aws
input_parameters = [
  {
    "parameter_name" = "AccessKeysRotatedParameterMaxAccessKeyAge"
    "parameter_value" = "90"
  },
]
region_name = us-east-1
resource_aka = arn:aws:config:us-east-1:352685002396:conformance-pack/turbottest37108/conformance-pack-pmqlupbus
resource_name = turbottest37108

Running SQL query: test-get-query.sql
[
  {
    "akas": [
      "arn:aws:config:us-east-1:352685002396:conformance-pack/turbottest37108/conformance-pack-pmqlupbus"
    ],
    "delivery_s3_bucket": "<null>",
    "delivery_s3_key_prefix": "<null>",
    "input_parameters": [
      {
        "ParameterName": "AccessKeysRotatedParameterMaxAccessKeyAge",
        "ParameterValue": "90"
      }
    ],
    "name": "turbottest37108",
    "title": "turbottest37108"
  }
]
✔ PASSED

Running SQL query: test-list-call-query.sql
[
  {
    "input_parameters": [
      {
        "ParameterName": "AccessKeysRotatedParameterMaxAccessKeyAge",
        "ParameterValue": "90"
      }
    ],
    "name": "turbottest37108",
    "title": "turbottest37108"
  }
]
✔ PASSED

Running SQL query: test-not-found-query.sql
null
✔ PASSED

Running SQL query: test-turbot-query.sql
[
  {
    "account_id": "352685002396",
    "akas": [
      "arn:aws:config:us-east-1:352685002396:conformance-pack/turbottest37108/conformance-pack-pmqlupbus"
    ],
    "name": "turbottest37108",
    "region": "us-east-1",
    "title": "turbottest37108"
  }
]
✔ PASSED

POSTTEST: tests/aws_config_conformance_pack

TEARDOWN: tests/aws_config_conformance_pack

SUMMARY:

1/1 passed.
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
> select
  name,
  conformance_pack_id,
  created_by,
  last_update_requested_time,
  title,
  akas
from
  aws_config_conformance_pack;
+----------+----------------------------+------------+----------------------------+----------+----------------------------------------------------------------
| name     | conformance_pack_id        | created_by | last_update_requested_time | title    | akas                                                           
+----------+----------------------------+------------+----------------------------+----------+----------------------------------------------------------------
| test-001 | conformance-pack-sodcgo3zh | <null>     | 2021-03-23 07:38:22        | test-001 | ["arn:aws:config:us-east-1:352685002396:conformance-pack/test-0
+----------+----------------------------+------------+----------------------------+----------+----------------------------------------------------------------

> select
  name,
  inp ->> 'ParameterName' as parameter_name,
  inp ->> 'ParameterValue' as parameter_value,
  title,
  akas
from
  aws_config_conformance_pack,
  jsonb_array_elements(input_parameters) as inp;
+----------+----------------+-----------------+----------+------------------------------------------------------------------------------------------------+
| name     | parameter_name | parameter_value | title    | akas                                                                                           |
+----------+----------------+-----------------+----------+------------------------------------------------------------------------------------------------+
| test-001 | ID             | 001             | test-001 | ["arn:aws:config:us-east-1:352685002396:conformance-pack/test-001/conformance-pack-sodcgo3zh"] |
+----------+----------------+-----------------+----------+------------------------------------------------------------------------------------------------+

> select
  name,
  conformance_pack_id,
  delivery_s3_bucket,
  delivery_s3_key_prefix
from
  aws_config_conformance_pack;
+----------+----------------------------+--------------------+------------------------+
| name     | conformance_pack_id        | delivery_s3_bucket | delivery_s3_key_prefix |
+----------+----------------------------+--------------------+------------------------+
| test-001 | conformance-pack-sodcgo3zh | <null>             | <null>                 |
+----------+----------------------------+--------------------+------------------------+
```
</details>

### CODE REVIEW

File: aws/table_aws_config_conformance_pack.go [hunk 1]
```diff
@@ -24,55 +24,48 @@ func tableAwsConfigConformancePack(_ context.Context) *plugin.Table {
...
 			{
 				Name:        "name",
 				Description: "Name of the conformance pack.",
 				Type:        proto.ColumnType_STRING,
 				Transform:   transform.FromField("ConformancePackName"),
 			},
 			{
-				Name:        "conformance_pack_id",
-				Description: "ID of the conformance pack.",
+				Name:        "conformance_pack_arn",
```
Comment (cbruno10) [inline aws/table_aws_config_conformance_pack.go:34]: Following Terraform, we should keep this as `arn` (https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/config_conformance_pack#arn). Also, speaking with @johnsmyth , we normally do name any ARN fields as `arn` (we should clarify this in our [developer docs for naming](https://steampipe.io/docs/develop/standards) though).

File: aws/table_aws_config_conformance_pack.go [hunk 2]
```diff
@@ -24,55 +24,48 @@ func tableAwsConfigConformancePack(_ context.Context) *plugin.Table {
...
-				Transform:   transform.FromField("DeliveryS3KeyPrefix"),
 			},
 			{
 				Name:        "last_update_requested_time",
 				Description: "Last update to the conformance pack.",
 				Type:        proto.ColumnType_TIMESTAMP,
-				Transform:   transform.FromField("LastUpdateRequestedTime"),
 			},
+			{
+				Name:        "conformance_pack_input_parameters",
```
Comment (cbruno10) [inline aws/table_aws_config_conformance_pack.go:64]: For this one also, following Terraform, I believe `input_parameters` is the name to use. Terraform uses the name `input_parameter` due to limitations while creating a conformance pack, so our name is in between Go SDK (plural) and Terraform (dropping `conformance_pack_`), resulting in `input_parameters`.

File: docs/tables/aws_config_conformance_pack.md [hunk 3]
```diff
@@ -7,17 +7,41 @@ A conformance pack is a collection of AWS Config rules and remediation actions t
...
+  akas
+from
+  aws_config_conformance_pack;
 ```
 
+
+### Input parameter details of each conformance pack
+
+```sql
+select
```
Comment (cbruno10) [inline docs/tables/aws_config_conformance_pack.md:25]: What type of information is contained in the input parameters field?

**Review by cbruno10 — CHANGES_REQUESTED** [review]
Please see review comments, thanks!

File: docs/tables/aws_config_conformance_pack.md (hunk 3 above)
Comment (Paulami30) [reply docs/tables/aws_config_conformance_pack.md:25]: It contains Input parameters in the form of key-value pairs for the conformance pack -`ParameterName`  and `ParameterValue`

**Comment by cbruno10** [comment]
@Paulami30 After updating the column names, can you please re-run the integration test and update the results in the PR's original message? I still see some references to `conformance_pack_input_parameters` in the SQL queries for instance.

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**